drivers/SmartThings/matter-switch/src/init.lua

```diff
@@ -559,7 +559,7 @@ local function initialize_switch(driver, device)
     else
       configure_buttons(device)
     end
-  elseif #switch_eps > 0 then
+  else
```

Comment:
I would still recommend explicitly making this case check for the switch_eps, just that is it clear that this is meant to handle the switch case rather than just being a catch-all case. Theoretically, this logic should behave the same either way with how our driver is currently setup, but I think still trying it explicitly to the switch eps makes it less prone to edge cases.

Reply:
Sure thing, added it back!